config.ini

```diff
@@ -28,27 +28,27 @@
 #redis_db = 0
 
 ## Secret used to sign forwarded requests
-#secret = 
+secret = super_secret_jira_key
```

Comment:
I don't think these values are good as the defaults. IMHO there are no such good values and so I prefer to have this file semantically empty.